### PR TITLE
Make Renovate bot ignore forks and remove the SENTR_DSN

### LINKED ISSUE #1625: Remove Sentry DSN from repository
## Problem

If someone configures their local environment correctly, they can send scary (and hard to discern) errors to Sentry that we can easily mistake for production issues. It's not likely that anyone is doing this intentionally (what do they stand to gain, other than trolling, I suppose) but it is easy to do accidentally.

## Description

Let's just remove the Sentry DSN from the repository. The environment variable can default to empty and Sentry just won't be enabled in that case. In production and staging we'll pass the Sentry DSN like we do any other secret. Let's go ahead and rotate the DSN as well once we deploy this change.

## Alternatives
Make it harder to accidentally (or intentionally) configure Sentry to run locally. Maybe Sentry has an optional password or something.

## Additional context
Marked as Staff only as it requires infrastructural changes to be completed.

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

### LINKED ISSUE #1650: Disable Renovate bot to run on forks
## Description

A contributor communicated receiving an influx of emails of failed Renovate tasks, so because of this we want to explicitly disable it on forks, although the [docs](https://docs.renovatebot.com/configuration-options/#includeforks) say that should be the default behavior.

## Additional context
[Slack thread](https://wordpress.slack.com/archives/C02012JB00N/p1660208114397979) where was reported.

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by krysal**
## Fixes

Fixes #1625 by @sarayourfriend 
Fixes #1650 by @krysal 

## Description

This PR:

- makes Renovate bot ignore forks
- removes Openverse' Sentry DNS and disable it if it is not set

## Testing Instructions

For Sentry, try running the project with different environment variables configurations.

1. Simulating production and without setting the SENTRY_DSN, confirm it's disabled
```sh
NODE_ENV=production pnpm dev:only
```
In your terminal you should see an output like this:
```sh
✔ Sentry reporting is disabled ("disabled" option has been set)                                                       nuxt:sentry 16:13:45

   ╭─────────────────────────────────────────────╮
   │                                             │
   │   Nuxt @ v2.15.8                            │
   │                                             │
   │   ▸ Environment: development (production)   │
   │   ▸ Rendering:   server-side                │
   │   ▸ Target:      server                     │
   │                                             │
   │   Listening: http://172.16.0.12:8443/       │
   │                                             │
   ╰─────────────────────────────────────────────╯
```

2. With production + a SENTRY_DSN string, it should be **enabled**
```sh
NODE_ENV=production SENTRY_DSN=<redacted> pnpm dev:only
```

3. For `development` it should remain **disabled**

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [N/A] I added or updated tests for the changes I made (if applicable).
- [N/A] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: .github/renovate.json [hunk 1]
```diff
@@ -24,5 +24,6 @@
     ".github/workflows/pr_label_check.yml",
     ".github/workflows/pr_ping.yml",
     "package.json"
-  ]
+  ],
+  "includeForks": false
```
Comment (sarayourfriend) [inline .github/renovate.json:28]: Nice!

**Review by sarayourfriend — APPROVED** [review]
This LGTM :rocket: Can you also open a PR in the infrastructure repository to add the `SENTRY_DSN` key to the ECS frontend module using a new DSN for the frontend Sentry project?

**Review by zackkrida — APPROVED** [review]
lgtm

**Comment by krysal** [comment]
@sarayourfriend I'll do it! I was preparing this work for that 😄 Though I'll have to apply it on Monday given today is like Friday for me (I'll be AFK tomorrow).

Thank you for the quick review ⚡️

**Comment by sarayourfriend** [comment]
Perfect :slightly_smiling_face:

**Pull request merged**